game/player.h

```diff
@@ -0,0 +1,33 @@
+#ifndef PLAYER_H_
+#define PLAYER_H_
+
+#include <vector>
+#include <memory>
+#include <utility>
+
+#include "hackmon.h"
+#include "statInfo.h"
+
+class Player {
+ public:
+  std::string name;
+  std::vector<unique_ptr<Hackmon>> party;
+  int winTotal;
+  int nextAlive;
```

Comment:
`winTotal` and `nextAlive` should probably be unsigned

Reply:
This is also true for the parameters to `isAlive`, `getHackmon`, and `swapHackmon`